lib/src/key/mod.rs

```diff
@@ -36,6 +38,7 @@
 /// crate::key::table::ft                /*{ns}*{db}*{tb}!ft{ft}
 /// crate::key::table::ix                /*{ns}*{db}*{tb}!ix{ix}
 /// crate::key::table::lq                /*{ns}*{db}*{tb}!lq{lq}
+/// crate::key::table::tb                /*{ns}*{db}*{tb}!tb [new]
```

Comment:
This is basically https://github.com/surrealdb/surrealdb/pull/2554/files#r1311674064 for tbs.